src/main/java/com/lb/application/BookingAccommodation.java

```diff
@@ -0,0 +1,35 @@
+package com.lb.application;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.ZonedDateTime;
+import java.util.List;
+
+public record BookingAccommodation(
+    String id,
+    String name,
+    String neighborhood,
+    ZonedDateTime checkin,
+    ZonedDateTime checkout,
+    int pricepernight) {
+
+  private static final Logger log = LoggerFactory.getLogger(BookingAccommodation.class);
+
+  public static List<BookingAccommodation> extract(InputStream json) {
+    ObjectMapper mapper = new ObjectMapper();
+    mapper.registerModule(new JavaTimeModule());
```

Comment:
use `JsonSupport.getObjectMapper()` from the Akka sdk, it already has several relevant modules registered

Reply:
removed